src/Microsoft.Health.SqlServer.Api/Controllers/SchemaController.cs

```diff
@@ -44,11 +49,11 @@ public ActionResult AvailableVersions()
             _logger.LogInformation("Attempting to get available schemas");
 
             var availableSchemas = new List<object>();
-            var currentVersion = _schemaInformation.Current ?? 0;
-            foreach (var version in Enum.GetValues(typeof(SchemaVersion)).Cast<SchemaVersion>().Where(sv => sv >= currentVersion))
+            var currentVersion = _schemaInformation.Current ?? 1;
```

Comment:
is this intentional change?

Reply:
Yes, because we no longer have access to the enum in this class, we need to initialize with 1 since we're just using the int iterator to build the data.